azurelinuxagent/common/AgentGlobals.py

```diff
@@ -17,6 +17,60 @@
 # Requires Python 2.6+ and Openssl 1.0+
 
 
+# too-few-public-methods<R0903> Disabled: This class is used as an Enum
+class FeatureNames(object):  # pylint: disable=R0903
+    """
+    Enum for defining the Feature Names for all internal and CRP features
+    """
+    MultiConfig = "MultipleExtensionsPerHandler"
+    ExtensionTelemetryPipeline = "ExtensionTelemetryPipeline"
+
+
+class AgentFeature(object):
```

Comment:
we should move this class to its own file

we also need to clarify its intent (and maybe rename it?) "new features that the Linux Guest Agent supports" seems too vague

should we use this only for the features were CRP needs to adjust its behavior depending on what the agent supports?

Reply:
Sure, any suggestions? 
CrpSupportedAgentFeature? Interface for defining all features that the Linux Guest Agent supports and needs to report back to CRP?

Reply:
SupportedFeature? or just clarify its intent in the doctring

Reply:
Done